src/main/java/sc/fiji/colourDeconvolution/ColorDeconvolutionIJ2.java

```diff
@@ -67,8 +68,11 @@ public ColorDeconvolutionIJ2() {
     public void run() {
         StainMatrixIJ2 sm = new StainMatrixIJ2();
         sm.init("User defined stain", R1, G1, B1, R2, G2, B2, R3, G3, B3);
+        final ImgPlus<? extends RealType<?>> imgPlus = dataset.getImgPlus();
+
+        imgPlus.factory().type();
```

Comment:
Is this necessary for proper initialisation? It looks useless...

Reply:
That is one thing for testing I missed that. I will remove it.